### PR TITLE
feat(content): minor improvements

### PR DESCRIPTION
**Pull request by segmentJason**
### Proposed changes
- minor improvements to grammar while reviewing

### Merge timing
- ASAP once approved?

### CODE REVIEW

File: src/connections/destinations/catalog/marketo-static-lists/index.md [hunk 1]
```diff
@@ -127,4 +127,4 @@ You might need to adjust your query to filter out users without an email so you
 
 ![](images/personas-add-emailtrait.png)
 
-A user with multiple email addresses appears once per email address in the Marketo lists.
+A user with multiple email addresses will appear once per email address in the Marketo lists.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/marketo-static-lists/index.md:130]: @segmentJason what does this sentence mean? Is this saying that a user that has multiple email addresses has each email appear once in the Marketo lists?

Comment (segmentJason) [reply src/connections/destinations/catalog/marketo-static-lists/index.md:130]: @stayseesong Yes, that is what it points out. Maybe an unnecessary change. "addresses appears" seemed awkward to read 😄

Comment (stayseesong) [reply src/connections/destinations/catalog/marketo-static-lists/index.md:130]:
```suggestion
If a user has multiple email addresses, each address appears once in the Marketo lists. 
```

Comment (stayseesong) [reply src/connections/destinations/catalog/marketo-static-lists/index.md:130]: @segmentJason how's this☝️

Comment (segmentJason) [reply src/connections/destinations/catalog/marketo-static-lists/index.md:130]: Works for me! Thanks @stayseesong

**Review by stayseesong — APPROVED** [review]

**Pull request merged**